apps/server/src/controllers/ontimeController.ts

```diff
@@ -207,6 +207,11 @@ export const postSettings = async (req, res) => {
     const editorKey = extractPin(req.body?.editorKey, settings.editorKey);
     const operatorKey = extractPin(req.body?.operatorKey, settings.operatorKey);
 
+    if (isDocker && req.body?.serverPort) {
+      return res.status(403).json({ message: `Can't change port when running inside docker` });
+    }
+    const serverPort = parseInt(req.body?.serverPort ?? settings.serverPort, 10);
```

Comment:
I usually prefer using the casting `Number(value)`.
I guess in this case it would have been validated before getting here, but it cant hurt

Reply:
But logically port value can't be fractional. Hence `parseInt` makes more sense i think.